content/docs/actors/development/secret_input.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Secret input
+description: Learn about making some input fields secret and encrypted.
+paths:
+# NOTE: IF ADDING A NEW PATH, LEAVE THE OLD ONES FOR REDIRECTS
+    - actor/secret-input
+    - actors/secret-input
+    - actor/development/secret-input
+    - actors/development/secret-input
+---
+
+# [](#secret-input)Secret input
+
+The Secret Input feature allows you to make an input field secret, causing it to be stored encrypted when setting the actor input, and decrypted only inside the actor.
```

Comment:
Prolly no need for the caps here. Also a lil readability suggestion, but just a suggestion 
```suggestion
The secret input feature allows you to encrypt sensitive actor inputs such as API keys and passwords. The input is then decrypted inside the actor.
```

Reply:
I wanted to be explicit here, to say when does the encryption and decryption happen. But I'll adjust it a bit.